w2/sym.py

```diff
@@ -1,7 +1,9 @@
-class SYM:
+import math
+
+class Sym:
```

Comment:
SYM?

Reply:
Yeah. I will update it with the next commit.